src/ImageSharp/Formats/Webp/Lossless/Vp8LHashChain.cs

```diff
@@ -280,5 +287,25 @@ private static int GetWindowSizeForHashChain(int quality, int xSize)
 
             return maxWindowSize > WindowSize ? WindowSize : maxWindowSize;
         }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!this.disposed)
+            {
+                if (disposing)
+                {
+                    this.OffsetLength.Dispose();
+                }
+
+                this.disposed = true;
+            }
+        }
+
+        /// <inheritdoc />
+        public void Dispose()
+        {
+            this.Dispose(disposing: true);
+            GC.SuppressFinalize(this);
```

Comment:
If your types can be sealed I would do so and avoid `SuppressFinalize` where it is not necessary.

Reply:
Yeah I think the `Dispose(bool disposing)` pattern is only favorable when we actually have a finalizer or expect subclasses to have a finalizer.